app/controllers/admin/organizations_controller.rb

```diff
@@ -6,10 +6,11 @@ def edit
 
   def update
     @organization = Organization.find(params[:id])
+
     if OrganizationUpdateService.update(@organization, organization_params)
       redirect_to admin_organizations_path, notice: "Updated organization!"
     else
-      flash[:error] = "Failed to update your organization."
+      flash[:error] = @organization.errors.full_messages.first
```

Comment:
Is there a reason you're only grabbing the first error and not all of them? `@organization.errors.full_messages.join("\n")`